### ISSUE TITLE
settings/parameters dialog close button has no accessible name

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3932]
**Issue Description**:
The settings/parameters dialog does not have an accessible name for the "X" button used to close the menu. 

WCAG Criteria:
[SC 4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

Screenshot or Screen Recording:
<img width="1603" alt="Screenshot 2024-09-05 at 5 23 25 PM" src="https://github.com/user-attachments/assets/002b83a7-9c13-4a8c-b52a-93bc544600c1">

Notes: can use aria-label to give the button an accessible name. "close" should be the name because it will be clear from the context that it is intended to close the dialog that opens.

**Comment by danny-avila** [comment]
Closing as this UI element is now deprecated
